### PR TITLE
Changes after code revision

### PR DESCRIPTION
**Pull request by felp99**

### CODE REVIEW

**Comment by felp99** [comment]
Related: [https://github.com/astrolabe-expeditions/data-platform/pull/4](https://github.com/astrolabe-expeditions/data-platform/pull/4)

**Review by cballevre — APPROVED** [review]

**Comment by cballevre** [comment]
Before to merge you need to make a commit to add `yarn add @radix-ui/themes` and rebase onto my modification I added a key also to deploy the app as it was block by the linter

File: app/[locale]/(dashboard)/[id]/page.jsx
Comment (cballevre) [inline app/[locale]/(dashboard)/[id]/page.jsx:1]: I think we can delete this page

File: app/[locale]/(dashboard)/sensors/[id]/page.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  if (!session) {
+    redirect('/auth/login')
+  }
+
+  return (
+    <Page>
+      <IconButton icon={Left} variant="secondary">
+        <Link href={'/sensors'}></Link>
+      </IconButton>
```
Comment (cballevre) [inline app/[locale]/(dashboard)/sensors/[id]/page.jsx:20]: We can add the back button as a part of the PageHeader

File: app/[locale]/(dashboard)/sensors/[id]/edit/page.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { getServerSession } from 'next-auth/next'
+import { authOptions } from '@/lib/auth'
+import { Page } from '@/components/Page/Page'
+import { PageHeader } from '@/components/Page/PageHeader'
+
+async function Home({ params }) {
```
Comment (cballevre) [inline app/[locale]/(dashboard)/sensors/[id]/edit/page.jsx:6]: By convention, I think we can call the component by name related with the page route like `SensorEdit`

File: app/[locale]/layout.jsx [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import { I18nProvider } from '@/components/I18nProvider'
 import '@/styles/global.css'
+import '@radix-ui/themes/styles.css'
```
Comment (cballevre) [inline app/[locale]/layout.jsx:3]: This should be added into the file `global.css`

File: app/[locale]/layout.jsx [hunk 4]
```diff
@@ -13,7 +15,9 @@ const RootLayout = ({ children }) => {
   return (
     <html lang={locale}>
       <body>
-        <I18nProvider locale={locale}>{children}</I18nProvider>
+        <Theme>
```
Comment (cballevre) [inline app/[locale]/layout.jsx:18]: I think we can create a `Providers` component to put all client side provider in the same place

File: components/Sidebar/Sidebar.jsx [hunk 5]
```diff
@@ -8,7 +8,8 @@ const Sidebar = () => {
       <SidebarLogo />
       <nav>
         <ul className="flex flex-col gap-1">
-          <SidebarItem name="Stations" to="/" />
+          <SidebarItem name="Dashboard" to="/" />
```
Comment (cballevre) [inline components/Sidebar/Sidebar.jsx:11]: I need to show you how to extract all the string into translation file so we can have a french version

File: components/ui/Icons/EllipsisVertical.svg
Comment (cballevre) [inline components/ui/Icons/EllipsisVertical.svg:1]: I think the DotVertical icon is kind of the same maybe ?

File: app/[locale]/(dashboard)/sensors/[id]/page.jsx (hunk 1 above)
Comment (felp99) [reply app/[locale]/(dashboard)/sensors/[id]/page.jsx:20]: I did that. Doing the PR today!

Comment (cballevre) [reply app/[locale]/(dashboard)/sensors/[id]/page.jsx:20]: Cool but merge this PR before doing new change

File: app/[locale]/layout.jsx (hunk 4 above)
Comment (felp99) [reply app/[locale]/layout.jsx:18]: Like the Theme, right?

File: components/ui/Icons/EllipsisVertical.svg
Comment (felp99) [reply components/ui/Icons/EllipsisVertical.svg:1]: Yes, I did not see it 🙃

**Comment by cballevre** [comment]
I'm closing pull request because all the changes has been merge with this PR : https://github.com/astrolabe-expeditions/data-platform/pull/12

**Pull request closed without merging**